### ISSUE TITLE
Accuracy

### ISSUE THREAD

**Issue by Ysmita** [klickmal/ContextNet#4]
Hi @klickmal ,

I trained the network for 10 epochs and obtained train accuracy of 82% and validation accuracy of 81% but still the images are blur. Is this because of insufficient training?
